src/condition.cpp

```diff
@@ -928,7 +929,7 @@ bool ConditionRegeneration::executeCondition(Creature* creature, int32_t interva
 	if (internalManaTicks >= manaTicks) {
 		internalManaTicks = 0;
 
-		if (Player* player = creature->getPlayer()) {
+		if (auto player = creature->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
		if (const auto player = creature->getPlayer()) {
```